cdc/scheduler/schedule_dispatcher.go

```diff
@@ -445,7 +447,8 @@ func (s *BaseScheduleDispatcher) removeTable(
 	// need to delete table
 	captureID := record.CaptureID
 	epoch := s.captureStatus[captureID].Epoch
-	ok, err = s.communicator.DispatchTable(ctx, s.changeFeedID, tableID, captureID, true, epoch)
+	ok, err = s.communicator.DispatchTable(ctx, s.changeFeedID, tableID,
```

Comment:
Does it have to be 0 here?

Reply:
remove don't need startTs or checkpointTs, so it can be any value.